### PR TITLE
v1 doc updates for the following API's

### PR DESCRIPTION
**Pull request by svpsiva**
v1 updates for the following API's
1. Message reply API
2. Mailbox Settings API

### CODE REVIEW

File: api-reference/v1.0/api/message-reply.md [hunk 1]
```diff
@@ -76,32 +103,29 @@ Content-length: 32
 [!INCLUDE [sample-code](../includes/snippets/objc/message-reply-objc-snippets.md)]
 [!INCLUDE [sdk-documentation](../includes/snippets/snippets-sdk-documentation-link.md)]
 
-# [Java](#tab/java)
```
Comment (angelgolfer-ms) [inline api-reference/v1.0/api/message-reply.md]: @svpsiva , was there a reason you deleted the lines for Java?
In general, we should just leave the lines for each language alone. There is a regular cadence when these lines will get regenerated appropriately, after we change the REST example.

File: api-reference/v1.0/api/message-reply.md [hunk 2]
```diff
@@ -76,32 +103,29 @@ Content-length: 32
...
-##### Response
 ##### Response
 Here is an example of the response.
 <!-- {
   "blockType": "response",
   "truncated": true
 } -->
 ```http
-HTTP/1.1 200 OK
+HTTP/1.1 201 Created
```
Comment (angelgolfer-ms) [inline api-reference/v1.0/api/message-reply.md]: @svpsiva , the "Response" section says this returns HTTP 202 Accepted. 
Which response code is intended?

File: api-reference/v1.0/api/user-get-mailboxsettings.md [hunk 3]
```diff
@@ -28,30 +37,36 @@ One of the following permissions is required to call this API. To learn more, in
...
-To get all mailbox settings for a user:
+To get all the mailbox settings for a user:
 <!-- { "blockType": "ignored" } -->
 ```http
 GET /me/mailboxSettings
 GET /users/{id|userPrincipalName}/mailboxSettings
 ```
 
-To get specific settings - for example, only the automatic replies settings, locale, time zone, or working hours:
+To get specific settings - only the automatic replies settings, locale, time zone, or working hours:
```
Comment (angelgolfer-ms) [inline api-reference/v1.0/api/user-get-mailboxsettings.md]: Oh, if we mean to list all the individual settings, then line 47 is missing date format and time format.
Same in the beta version of this file.

File: api-reference/v1.0/api/user-get-mailboxsettings.md [hunk 4]
```diff
@@ -95,14 +114,10 @@ GET https://graph.microsoft.com/v1.0/me/mailboxSettings
 [!INCLUDE [sample-code](../includes/snippets/objc/get-mailboxsettings-1-objc-snippets.md)]
 [!INCLUDE [sdk-documentation](../includes/snippets/snippets-sdk-documentation-link.md)]
 
-# [Java](#tab/java)
```
Comment (angelgolfer-ms) [inline api-reference/v1.0/api/user-get-mailboxsettings.md]: Was there a reason you deleted the lines for Java?
We can just leave these language-specific lines alone. They will get regenerated when we update the REST example.

File: api-reference/v1.0/api/user-get-mailboxsettings.md [hunk 5]
```diff
@@ -175,13 +193,9 @@ GET https://graph.microsoft.com/v1.0/me/mailboxSettings/automaticRepliesSetting
 [!INCLUDE [sample-code](../includes/snippets/objc/get-mailboxsettings-2-objc-snippets.md)]
 [!INCLUDE [sdk-documentation](../includes/snippets/snippets-sdk-documentation-link.md)]
 
-# [Java](#tab/java)
```
Comment (angelgolfer-ms) [inline api-reference/v1.0/api/user-get-mailboxsettings.md]: Same comment - we can just leave the Java lines alone and let them get regenerated automatically.

File: api-reference/v1.0/api/user-update-mailboxsettings.md [hunk 6]
```diff
@@ -128,7 +135,7 @@ HTTP/1.1 200 OK
 Content-type: application/json
 
 {
-    "@odata.context": "https://graph.microsoft.com/v1.0/$metadata#Me/mailboxSettings",
+    "@odata.context": "https://graph.microsoft.com/beta/$metadata#Me/mailboxSettings",
```
Comment (angelgolfer-ms) [inline api-reference/v1.0/api/user-update-mailboxsettings.md]: This is the v1 topic, so line 138 should keep v1.0.

File: api-reference/v1.0/api/user-update-mailboxsettings.md [hunk 7]
```diff
@@ -156,7 +163,7 @@ to a [custom time zone](../resources/customtimezone.md).
   "name": "update_mailboxsettings_2"
 }-->
 ```http
-PATCH https://graph.microsoft.com/v1.0/me/mailboxSettings
+PATCH https://graph.microsoft.com/beta/me/mailboxSettings
```
Comment (angelgolfer-ms) [inline api-reference/v1.0/api/user-update-mailboxsettings.md]: Line 166 - keep v1.0.

File: api-reference/v1.0/api/user-update-mailboxsettings.md [hunk 8]
```diff
@@ -206,7 +213,7 @@ HTTP/1.1 200 OK
 Content-type: application/json
 
 {
-    "@odata.context":"https://graph.microsoft.com/v1.0/$metadata#users('94447c6e-ea4c-494c-a9ed-d905e366c5cb')/mailboxSettings",
+    "@odata.context":"https://graph.microsoft.com/beta/$metadata#users('94447c6e-ea4c-494c-a9ed-d905e366c5cb')/mailboxSettings",
```
Comment (angelgolfer-ms) [inline api-reference/v1.0/api/user-update-mailboxsettings.md]: Line 216, keep v1.0.

File: api-reference/v1.0/api/message-reply.md (hunk 2 above)
Comment (angelgolfer-ms) [reply api-reference/v1.0/api/message-reply.md]: While waiting for your confirmation, I changed it to `202 Accepted` for consistency.

File: api-reference/v1.0/api/user-get-mailboxsettings.md (hunk 3 above)
Comment (angelgolfer-ms) [reply api-reference/v1.0/api/user-get-mailboxsettings.md]: Fixed to enumerate all.

File: api-reference/v1.0/api/user-update-mailboxsettings.md (hunk 6 above)
Comment (angelgolfer-ms) [reply api-reference/v1.0/api/user-update-mailboxsettings.md]: Fixed.

File: api-reference/v1.0/api/user-update-mailboxsettings.md (hunk 7 above)
Comment (angelgolfer-ms) [reply api-reference/v1.0/api/user-update-mailboxsettings.md]: Fixed.

File: api-reference/v1.0/api/user-update-mailboxsettings.md (hunk 8 above)
Comment (angelgolfer-ms) [reply api-reference/v1.0/api/user-update-mailboxsettings.md]: Fixed.

File: api-reference/v1.0/api/message-reply.md (hunk 1 above)
Comment (svpsiva) [reply api-reference/v1.0/api/message-reply.md]: This was the case in beta files. I just copied the lines. We can readd.

File: api-reference/v1.0/api/user-get-mailboxsettings.md (hunk 4 above)
Comment (svpsiva) [reply api-reference/v1.0/api/user-get-mailboxsettings.md]: No specific reason. We can readd.

File: api-reference/v1.0/api/user-update-mailboxsettings.md (hunk 6 above)
Comment (svpsiva) [reply api-reference/v1.0/api/user-update-mailboxsettings.md]: Makes sense.

**Pull request merged**